src/DebugApi.ts

```diff
@@ -199,13 +199,15 @@ export class DebugApi {
         };
         const server = query["server"] as string;
         const since = parseInt(query["since"] as string);
+        const limit = query["limit"] !== undefined ? parseInt(query["limit"] as string) : undefined;
```

Comment:
`limit` as in maxKickedUsers?
The term limit could apply to many different restrictions and I wish the naming was more specific.

Reply:
hmm, how about targetUserCount ?

Reply:
I chose limit specifically because it's easier to type into curl, tbh :p

Reply:
Have settled for `targetCount` which is shorter but more descriptive than `limit`.